content/en/platform/corda/4.6/open-source/release-notes.md

```diff
@@ -319,7 +319,7 @@ States and apps valid in Corda 3.0 and above are usable in Corda 4.5.
 
 #### Improved `killFlow` operations
 
-We have improved the existing [killFlow RPC operation](https://api.corda.net/api/corda-os/4.5/html/api/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/kill-flow.html), which allows node operators to terminate flows manually - in several ways:
+We have improved the existing [killFlow RPC operation](https://api.corda.net/api/corda-os/4.5/html/api/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/kill-flow.md), which allows node operators to terminate flows manually - in several ways:
```

Comment:
This one just redirects to the API site, but should link to this specific function.
```suggestion
We have improved the existing [killFlow RPC operation](../../../../../en/api-ref/corda/4.6/open-source/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/kill-flow.html), which allows node operators to terminate flows manually - in several ways:
```